google/cloud/storage/internal/insert_object_media_request.h

```diff
@@ -29,10 +29,11 @@ namespace internal {
  * TODO(#710) - add missing request parameters.
```

Comment:
I think you can remove the `TODO()`, right?  Search for other places where we say `#710` just in case I missed any.